draft-smyslov-ipsecme-ikev2-reliable-transport.xml

```diff
@@ -1,148 +1,220 @@
+      IKEv2 can operate over TCP, while IPsec tunnels use unreliable
+      transport. This feature allows IKEv2 to effectively exchange large blobs
+      of data (e.g., when post-quantum algorithms are employed) while avoiding
+      performance problems that arise when IPsec uses TCP.</t>
+    </abstract>
+  </front>
+
+  <middle>
+    <section anchor="intro" title="Introduction">
+      <t>The Internet Key Exchange protocol version 2 (IKEv2) <xref
+      target="RFC7296"></xref> originally used unreliable transport (UDP) for
+      its messages. Later it was extended to use TCP <xref
+      target="RFC9329"></xref> where UDP is blocked. UDP remains the preferred
+      transport for IKEv2, and TCP is only used if UDP datagrams cannot get
+      through.</t>
+
+      <t>Originally IKEv2 peers exchanged only a small amount of data, so that
+      simple retransmission mechanism on top of UDP with no congestion control
+      sufficed. The situation has changed when post-quantum cryptographic
+      (PQC) algorithms began to be incorporated into IKEv2 <xref
+      target="RFC9370"></xref>. Most of post-quantum algorithms require IKE
+      peers to exchange much more data, than classical algorithms, up to tens
+      (or even hundreds) Kbytes. A few proposals exist that allow to overcome
+      the 64 Kbytes limitation on the size of an IKE payload (<xref
+      target="I-D.nir-ipsecme-big-payload"></xref>, <xref
+      target="I-D.smyslov-ipsecme-ikev2-extended-pld"></xref>, <xref
+      target="I-D.tjhai-ikev2-beyond-64k-limit"></xref>).</t>
+
+      <t>When IKE messages grow to tens or even hundreds of kilobytes, using
+      UDP as a transport becomes challenging. The use of IKE fragmentation
+      <xref target="RFC7383"></xref> helps mitigate IP fragmentation issues
+      and ensures that each IKE message fragment fits into a UDP datagram,
+      even if the original message does not. However, all IKE fragments are
+      always sent (and retransmitted) simultaneously, meaning that as the
+      number of fragments increases and congestion control remains absent, the
+      simple retransmission mechanism of IKEv2 will perform poorly potentially
+      causing even more problems for the network.</t>
+
+      <t>Using reliable transport (e.g., TCP) for IKEv2 could be a solution to
+      the problem. However, the current use of TCP as defined in <xref
+      target="RFC9329"></xref> implies that ESP SAs are also encapsulated in
+      TCP, which has negative impact on IPsec performance (see Section 9 of
+      <xref target="RFC9329"></xref>).</t>
+
+      <t>This specification allows to decouple IKE and IPsec transports,
+      making it possible to use a reliable transport for IKEv2 while
+      continuing to use an unreliable transport for IPsec.</t>
+
+      <t>The idea to decouple IKE and IPsec transports was originally
+      presented in <xref
+      target="I-D.tjhai-ikev2-beyond-64k-limit"></xref>.</t>
```

Comment:
I believe this para should be at the end of this section.

Reply:
Fixed.